.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Build Docs
+on: push
+      
+permissions:
+  contents: write
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          auto-activate-base: false
+          miniconda-version: 'latest'
+          python-version: 3.11
```

Comment:
python 3.13 has been released recently. I think it's better to build the docs with the latest version of Python.

Reply:
changed.